cpp/src/io/csv/csv_gpu.cu

```diff
@@ -59,9 +59,8 @@ __device__ __inline__ bool is_whitespace(char c) { return c == '\t' || c == ' ';
  * @brief Scans a character stream within a range, and adjusts the start and end
  * indices of the range to ignore whitespace and quotation characters.
  *
- * @param data The character stream to scan
- * @param start The start index to adjust
- * @param end The end index to adjust
+ * @param[in] begin Beginning of the character string
```

Comment:
`trim_field_start_end`'s function arguments are not changed but documentation is updated.

Reply:
Ah, right. I forgot I hadn't updated this and did a find-replace. :facepalm: